keps/sig-node/4800-cpumanager-split-uncorecache/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: Split UnCoreCache Toplogy Awareness in CPU Manager
+kep-number: 4800
+authors:
+  - "@sphrasavath"
+  - "@wongchar"
+  - "@ajcaldelas"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: implementable 
+creation-date: 2024-08-23
+reviewers:
+  - "@ffromani"
+  - "@kannon92"
+  - "@kad"
+approvers:
+  - "@klueska" 
+  - "@sig-node-leads"
+
+see-also: 
+  - keps/sig-node/2625-cpumanager-policies-thread-placement"
+
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.32"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.32"
+  beta: "v1.33"
+  stable: "v1.35"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: "CPUManagerPolicyAlphaOptions"
+    components:
+      - kubelet
+disable-supported: true
+
+# The following PRR answers are required at beta release
+metrics: []
```

Comment:
This should be filled in with the metrics you've mentioned in a few places, not a blocker for alpha, though.

Reply:
If they don't actually exist yet, is that okay? We are still trying to come up with metrics to implement

Reply:
added container_aligned_compute_resource_count to metric field